analyzer/src/test/kotlin/managers/utils/MavenSupportTest.kt

```diff
@@ -140,6 +140,36 @@ class MavenSupportTest : WordSpec({
                 path = "boringssl-static"
             )
         }
+
+        "handle GitHub URLs with double 'git:' prefix" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    connection = "scm:git:git:github.com/MarkusAmshove/Kluent.git"
+                }
+            }
+
+            MavenSupport.parseVcsInfo(mavenProject) shouldBe VcsInfo(
+                type = VcsType.GIT,
+                url = "github.com/MarkusAmshove/Kluent.git",
+                revision = "",
+                path = ""
+            )
+        }
+
+        "handle GitHub URLs with missing 'git:' provider" {
+            val mavenProject = MavenProject().apply {
+                scm = Scm().apply {
+                    connection = "scm:git@github.com/Yalantis/uCrop.git"
+                }
+            }
+
+            MavenSupport.parseVcsInfo(mavenProject) shouldBe VcsInfo(
+                type = VcsType.GIT,
+                url = "https://github.com/Yalantis/uCrop.git",
```

Comment:
If we mapped it to `git@github.com/Yalantis/uCrop.git` IIUC `normalizeVcsUrl()` would add the `ssh` protocol. Is `https` intended here?

Reply:
> Is `https` intended here?

Sort of: While the `git@` username more indicates SSH, the `/` (instead of `:`) separator after the host more indicates HTTP(S). So basically both could be valid choices. However, for our read-only use-case HTTP(S) is more suitable as it does not enforce authentication.